ct.yaml

```diff
@@ -0,0 +1,9 @@
+# See https://github.com/helm/chart-testing#configuration
```

Comment:
Can we move this to `install/kubernetes` to avoid polluting root dir?

Reply:
agreed, let me send update for this.